src/test/scala/org/broadinstitute/dsde/workbench/sam/service/ResourceServiceSpec.scala

```diff
@@ -530,6 +530,56 @@ class ResourceServiceSpec extends AnyFlatSpec with Matchers with ScalaFutures wi
     policies should contain theSameElementsAs Set(expectedPolicy)
   }
 
+  it should "include memberPolicies in the policy list where applicable" in {
+    // create a "side" resource; we will use its policy emails as members in the default resource
+    val sideResource = FullyQualifiedResourceId(otherResourceType.name, ResourceId("side-resource"))
+    service.createResourceType(otherResourceType, samRequestContext).unsafeRunSync()
+    runAndWait(service.createResource(otherResourceType, sideResource.resourceId, dummyUserInfo, samRequestContext))
+    val sidePolicies = service.listResourcePolicies(sideResource, samRequestContext).unsafeRunSync()
+
+    // create the default resource
+    val resource = FullyQualifiedResourceId(defaultResourceType.name, ResourceId("my-resource"))
+    service.createResourceType(defaultResourceType, samRequestContext).unsafeRunSync()
+    runAndWait(service.createResource(defaultResourceType, resource.resourceId, dummyUserInfo, samRequestContext))
+
+    // add "side" policy emails to default resource
+    sidePolicies.foreach { sidePolicy =>
+      dirDAO.loadSubjectFromEmail(sidePolicy.email, samRequestContext).unsafeRunSync().map { subj =>
+        service.addSubjectToPolicy(FullyQualifiedPolicyId(resource, AccessPolicyName(defaultResourceType.ownerRoleName.value)), subj, samRequestContext).unsafeRunSync()
+      }
+    }
+
+    val actualPolicies = service.listResourcePolicies(resource, samRequestContext).unsafeRunSync()
+    actualPolicies.size shouldBe 1
+    val ownerPolicy = actualPolicies.head
+    ownerPolicy.policy.memberPolicies shouldBe defined
+    val actualMemberPolicies = ownerPolicy.policy.memberPolicies.get
+
+    val expectedMemberPolicies = sidePolicies.map { p =>
+      PolicyIdAndEmail(p.policyName, p.email, sideResource.resourceTypeName, sideResource.resourceId)
+    }
+
+    actualMemberPolicies shouldBe expectedMemberPolicies.toSet
```

Comment:
If this spec extends `OptionValues` then I think you can simplify this to be:

```suggestion
    ownerPolicy.policy.memberPolicies.value shouldBe expectedMemberPolicies.toSet
```
and then you can drop lines 555 and 556 as well.